src/components/form/delete.vue

```diff
@@ -21,6 +21,9 @@ except according to the terms contained in the LICENSE file.
           </template>
         </i18n-t>
         <p>{{ $t('introduction[1]') }}</p>
+        <p v-if="form.dataExists && form.entityRelated">
+          {{ $t('noDeleteEntities') }}
+        </p>
```

Comment:
I initially made this the second paragraph, but after looking at it, I think it actually works better as the last paragraph.